pkgs/by-name/cu/cunicu/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  protobuf,
+  protoc-gen-go,
+  protoc-gen-go-grpc,
+}:
+buildGoModule rec {
+  pname = "cunicu";
+  version = "0.5.53";
+
+  src = fetchFromGitHub {
+    owner = "cunicu";
+    repo = "cunicu";
+    rev = "v${version}";
+    hash = "sha256-y8JyXf+LdquhVr9G5wX1LdMF7wpj5PZG3xXGAWoypzA=";
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    protobuf
+    protoc-gen-go
+    protoc-gen-go-grpc
+  ];
+
+  CGO_ENABLED = 0;
+
+  vendorHash = "sha256-OiLVdEf6fcGHx0k0xC5sZwhnK0FiLgfdkz2zNgBbcgY=";
+
+  # These packages contain networking dependent tests which fail in the sandbox
+  excludedPackages = [
+    "pkg/config"
+    "pkg/selfupdate"
+    "pkg/tty"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.Version=${version}"
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.BuiltBy=Nix"
+  ];
+
+  preBuild = ''
+    go generate ./...
+  '';
+
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    cunicu docs --with-frontmatter
```

```suggestion
    $out/bin/cunicu docs --with-frontmatter
```

See https://github.com/NixOS/nixpkgs/pull/354164/checks?check_run_id=32744992651

---

Also below

---

Thanks, fixed in 5833f77de019